db/migrate/20210810091934_add_norm.rb

```diff
@@ -0,0 +1,20 @@
+# frozen_string_literal: true
+
+class AddNorm < ActiveRecord::Migration[6.1]
+  def up
+    create_table :norms do |t|
+      t.string :name
+
+      t.timestamps
+    end
+
+    add_column :freaks, :norm_id, :bigint
+    add_foreign_key :freaks, :norms
+  end
+
+  def down
+    remove_foreign_key :freaks, :norms
+    remove_column :freaks, :norm_id
```

Comment:
Doesn't removing the column remove the foreign key as well? :eyes:

Reply:
You are right :))